src/v2/ion/ViewClassNamesFactory.js

```diff
@@ -0,0 +1,88 @@
+/* eslint complexity: [2, 25] */
+import {FORMS} from './RemediationConstants';
+
+const FORMNAME_CLASSNAME_MAPPINGS = {
+  [FORMS.IDENTIFY]: {
+    'identify': ['identify', 'primary-auth'],
+    'password': ['identify', 'primary-auth']
+  },
+  [FORMS.ENROLL_PROFILE]: {
+    'enroll-profile': ['enroll-profile', 'registration'],
+  },
+  [FORMS.CHALLENGE_AUTHENTICATOR]: {
+    email: ['challenge-authenticator-email', 'mfa-verify-passcode'],
+    password: ['challenge-authenticator-password', 'mfa-verify-password'],
+    sms: ['challenge-authenticator-voice', 'mfa-verify-passcode'],
+    voice: ['challenge-authenticator-voice', 'mfa-verify-passcode'],
+    'security_question': ['challenge-authenticator-security_question', 'mfa-verify-question']
+  },
+  [FORMS.ENROLL_AUTHENTICATOR]: {
+    email: ['enroll-authenticator-email', 'enroll-email'],
+    password: ['enroll-authenticator-password', 'enroll-password'],
+    sms: ['enroll-authenticator-sms', 'enroll-sms'],
+    voice: ['challenge-authenticator-voice', 'mfa-verify-passcode'],
+    'security_question': ['enroll-authenticator-security_question', 'enroll-question'],
+  },
+  [FORMS.AUTHENTICATOR_ENROLLMENT_DATA]: {
```

Comment:
don't define it if there are no override.

Reply:
These cases still have an `authenticatorType` hence it goes into the if block and tries to evaluate `FORMNAME_CLASSNAME_MAPPINGS[formName][type]`

Reply:
- I probably meant base on the another comment that remove V2 classes from this mapping.
- Also, shall those class be `authenticator-enrollment-data-sms`, `authenticator-enrollment-data-voice` ?

Reply:
🤦  yup will update